ui/ozone/platform/wayland/wayland_keyboard.cc

```diff
@@ -108,8 +109,15 @@ void WaylandKeyboard::Key(void* data,
           dom_code, flags, &dom_key, &key_code))
     return;
 
-  // TODO(tonikitoo): handle repeat here.
   bool down = state == WL_KEYBOARD_KEY_STATE_PRESSED;
+
+  // TODO(tonikitoo): handle repeat here.
+  bool repeat = false;
+  if (!repeat) {
```

Comment:
I don't think this if conditional is needed in this cl at all.

Reply:
indeed. idea was for make it clear that we need to know whether we are handling key repeat of not here. This is similar to KeyboardEvDev::DispatchKey (https://cs.chromium.org/chromium/src/ui/events/ozone/evdev/keyboard_evdev.cc?q=keyboardevdev&sq=package:chromium&dr=CSs&l=237)

Reply:
The thing is that it doesn't sound like a proper commit for this even in this form. It will bring problems with fixups and rebasings. I would insist on removing it from here.

Reply:
fair enough. removed.